### PR TITLE
Separate the current .proto files into header and point files.

### PR DESCRIPTION
**Pull request by kberryUSGS**
Could someone take a look at this and make sure it looks like I've generally done the right thing in terms of git setup with this PR? If so, I'll go ahead and separate out the V001 .proto files and add this to the PR, then update.

### CODE REVIEW

**Comment by kberryUSGS** [comment]
[Separate the current .proto files into header and point files.](https://trello.com/c/K8AMsPHX/103-separate-the-current-proto-files-into-header-and-point-files)

**Comment by jessemapel** [comment]
It looks like you set everything up correctly

**Comment by kberryUSGS** [comment]
Okay, I updated this with the actual code now. Notably, V0001 was straightforward to separate into 2 .proto files for the Point and the Header, but V0001 wasn't set up that way. The two "top-level" (non-nested) messages were for `ControlNetLogDataProtoV0001` and `ControlNetFileProtoV0001`. (`PBControlMeasure` is inside of  `ControlNetFileProtoV0001.) So for V0002, rather than separating into header and point, I instead put each "top-level" message in its own file.

File: isis/src/control/objs/ControlNetVersioner/ControlNetFileEntryV0002.proto [hunk 1]
```diff
@@ -1,26 +1,17 @@
...
-  required string targetName   = 2;
-  optional string created      = 3;
-  optional string lastModified = 4;
-  optional string description  = 5;
-  optional string userName     = 6;
-  repeated int32  pointMessageSizes = 7 [packed = true];
-}
-
 message ControlPointFileEntryV0002 {
   enum PointType {
```
Comment (jessemapel) [inline isis/src/control/objs/ControlNetVersioner/ControlNetFileEntryV0002.proto]: Can we change the .proto file name to match the generated class? ControlNetFileEntryV0002.proto -> ControlPointFileEntryV0002.proto ?

File: isis/src/control/objs/ControlNetVersioner/ControlNetFileHeaderV0002.proto [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// 2011-06-07 Tracie Sucharski/Debbie Cook - Changed PointType 
+//                    Ground  ---->  Fixed
+//                    Tie     ---->  Free
```
Comment (jessemapel) [inline isis/src/control/objs/ControlNetVersioner/ControlNetFileHeaderV0002.proto]: This history comment only applies to points and can be removed from the Header .proto file

Comment (kberryUSGS) [reply isis/src/control/objs/ControlNetVersioner/ControlNetFileHeaderV0002.proto]: Good point, thanks!

File: isis/src/control/objs/ControlNetVersioner/ControlNetFileEntryV0002.proto (hunk 1 above)
Comment (kberryUSGS) [reply isis/src/control/objs/ControlNetVersioner/ControlNetFileEntryV0002.proto]: Thanks for catching that. It was indeed a typo.

**Comment by kberryUSGS** [comment]
@jessemapel Okay, I fixed those things.

**Pull request merged**